tests/xharness/Hardware/SimDevice.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Xharness;
+using Xharness.Execution;
+using Xharness.Logging;
+
+namespace xharness.Hardware {
```

Comment:
>Hardware [](start = 19, length = 8)

Not sure this namespace shouldn't be `Devices`. Not sure I would look for Simulator in Hardware?